src/backend/distributed/citus--6.2-2--6.2-3.sql

```diff
@@ -0,0 +1,12 @@
+/* citus--6.2-2--6.2-3.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION citus_running_version()
```

Comment:
Don't have this function. Use `citus.version` as a GUC instead, modeled after the `version` GUC in PostgreSQL.

It'll just return `6.1.0`, for instance.